index.js

```diff
@@ -78,6 +78,13 @@ bot.onText(/\/free/, (message) => {
       });
     });
 });
+
+bot.onText(/\/analyse/, async (message) => {
+  const userId = await _getUserId(message.chat.id);
+  const link = `http://localhost:3000/public?id=${userId}`;
```

Comment:
We can make this a link with mark up as follows:

const link = `<a href='http://127.0.0.1:3000/public?id=${userId}'>Analysis</a>`;
  bot.sendMessage(message.chat.id, link, {parse_mode: 'HTML'});

It seems like it doesn't handle localhost

As acquisitions.js relies on the URL we would need to change the BASE as follows

`const BASE = 'http://127.0.0.1:3000';
`

Then we can have a nice link from telegram.